iteration.qmd

```diff
@@ -168,7 +169,10 @@ df |>
 
 This is a little verbose, so R comes with a handy shortcut: for this sort of throw away, or **anonymous**[^iteration-1], function you can replace `function` with `\`:
 
-[^iteration-1]: Anonymous, because didn't give it a name with `<-.`
+[^iteration-1]: Anonymous, because we never explicitly created it and gave it a name with `<-`.
+    Another term programmers use for this is "lambda function".
```

Comment:
I like giving people this sort of background, i.e. de-mystifying jargon CS people love to throw around.